programs/survival/single_bedrock.sc

```diff
@@ -0,0 +1,82 @@
+global_DIM = 'the_nether';
+global_CHUNK_SIZE = 16;
+global_DRAW_RADIUS = 1;
+global_INTERVAL = 20;
+global_DURATION = global_INTERVAL + 2;
+global_TOP_Y = 127;
+global_CHECK_HEIGHT = 4;
+global_BLOCK_NAME = 'bedrock';
+global_DRAW_HEIGHT = 6;
+global_COLOR = 0x00ff0050;
+
+__config() -> {
+  'command_permission' -> 'players'
```

Comment:
Just out of curiosity, what would happen if you were to get rid of this line?

Reply:
I didn't test on a server. I think execute in server console would make `player()` return `null`. So I add this to prevent execute from console.